src/components/article-page.js

```diff
@@ -95,21 +95,86 @@ function getColumns(type) {
   }
 }
 
+const _fontLevel = {
+  base: 'base',
+  large: 'large',
+  xLarge: 'xLarge',
+}
+
 export default class Article extends PureComponent {
   static propTypes = {
     colors: predefinedPropTypes.colors,
     post: PropTypes.object.isRequired,
+    fontLevel: PropTypes.oneOf([
```

Comment:
Maybe the naming `defaultFontLevel` will be better since this prop is used to give the component a initial value  of `fontLevel` and then the component will hand out the control of `fontLevel` to the state of this component.